Embedding video link into sensitive data overview intro (#2850)
alan-turing-institute/the-turing-way: book/website/project-design/sdp/types.md

Therefore each research project needs to find the best solution for their needs.

The following sub-chapters give an overview of the types of sensitive data and metadata that need to be identified in datasets and therefore handled securely in research projects.

> Or you could watch this video below that has been created by ELIXIR-UK as an introduction to sensitive data.

Reviewer: ```suggestion
Here is a short video introducing sensitive data by ELIXIR-UK.
```

Reviewer: Looks good to me, https://deploy-preview-2850--the-turing-way.netlify.app/project-design/sdp/types.html.
